build.rs

```diff
@@ -221,9 +221,6 @@ fn check_php_version(info: &PHPInfo) -> Result<()> {
     // introduced in PHP 8.1).
     //
     // PHP 8.0 is the baseline - no feature flags will be introduced here.
-    //
-    // The PHP version cfg flags should also stack - if you compile on PHP 8.2 you
-    // should get both the `php81` and `php82` flags.
```

Comment:
Did you need to change this as part of this PR? This is quite a but change as it seems maybe you're changing the fall0through beahviour of `php81` being set on `php82` etc?

Reply:
I needed this to differentiate `php80` and `php81`. Guess i could have done a `and(php80, not(any(php81, php82, php83, php84)))` but that seemed a little unwieldy.

But I can pull this into a separate mr.

Reply:
yeah, I'd split out. I'd rather not making that change as part of this PR unless strictly necessary

Reply:
#345